pkg/controller/volume/attachdetach/populator/desired_state_of_world_populator.go

```diff
@@ -72,6 +88,18 @@ func (dswp *desiredStateOfWorldPopulator) Run(stopCh <-chan struct{}) {
 func (dswp *desiredStateOfWorldPopulator) populatorLoopFunc() func() {
 	return func() {
 		dswp.findAndRemoveDeletedPods()
+
+		// findAndAddActivePods is called every 3 minutes, independently of the main
```

Comment:
nit: Don't mention the exact time in the comment, it can change: `findAndAddActivePods is called every periodically...`